Source/BSN.Commons/Infrastructure/MessageBroker/RabbitMqEventAggregator/BrokerType.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace BSN.Commons.Infrastructure.MessageBroker.RabbitMqEventAggregator
+{
+    /// <summary>
+    /// Please refer to: <see href="https://medium.com/@bakhtmunir/rabbitmq-exchanges-in-c-8b3202fb3ab0"/>
+    /// </summary>
+    public enum BrokerType
+    {
+        /// <summary>
+        /// Direct exchange type routes messages based on exact matching of routing keys.
+        /// </summary>
+        direct,
```

Comment:
Why define enum value with little case?

By default we define enum value to CamelCase

Reply:
done.